viam_cartographer_test.go

```diff
@@ -148,22 +130,6 @@ func TestNew(t *testing.T) {
 		test.That(t, svc.Close(context.Background()), test.ShouldBeNil)
 	})
 
-	t.Run("Failed creation of cartographer slam service with invalid lidar sensor "+
```

Comment:
Removed this, because we're not validating in `CreateSLAMService` anymore whether or not the lidar or movement sensor is provides the data we expect from it.